templates/less/pages/signup.less

```diff
@@ -32,7 +34,7 @@
 
   div[role=form] {
     background: @white;
-    padding: 4rem;
+    padding: 25px;
```

Comment:
this is a thing that'll need to be reviewed soon, but rem for things that are dependent on text is generally better than pixels